qa/automation/CareerPortal.Tests.Scenarios/Burlaka.feature

```diff
@@ -0,0 +1,119 @@
+Then user should select "Other cookies" checkbox 
+And user should click "Submit preferences" button
+
+Scenario:1.2.3_07 Pop-up window "Cookies Settings" contains a link to "Data Privacy policy" pdf file
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
+When a pop-up window with cookies settings is displayed
+Then a link with "Data Privacy policy" text should be displayed
+
+Scenario Outline:1.2.3_08 Footer link buttons redirect a user to the corresponding websites
+Given user opened "career.quantori.com" 
+And user scrolled to the bottom of the page
+When user clicks a link button with a "<logo>" logo in the footer
+Then user should be redirected to "Quantori page" on the "<full website name>" website
+
+Examples:
+|logo                   |full website name     | 
+|"Planet icon"          |Quantori.com          | 
+|hh                     |hh.ru                 | 
+|f                      |facebook.com          | 
+|"Camera icon"          |instagram.com         |
+|vk                     |vk.com                |
+|in                     |linkedin.com          |
+
+Scenario:1.2.3_09 Footer link button does not redirect a user with Ukrainian IP address to Quantori page on vk.com website
+Given user did not change his Ukrainian IP address with VPN services 
+And user opened "career.quantori.com"
+And user scrolled to the bottom of the page
+When user clicks a link button with a "vk" logo in the footer
+Then user is not redirected to "Quantori page" on the "vk.com" website
+And message "There is no connection with the site" should be displayed 
+
+Scenario:1.2.3_10 Footer scroll-to-top button brings a user back to the top of the "career.quantori.com" page
+Given user opened "career.quantori.com"
+And user scrolled to the bottom of the page
+When user clicks a scroll-to-top button with a "QUANTORI" logo 
+Then user returns to the top of the "career.quantori.com" page
+And text "Become a part of Quantori team" should be displayed
+
+Scenario:1.2.3_11 Footer mailto button opens a new email window for a user, who signed in to his default email client account
+Given user signed in to his default email-client
+And user opened "career.quantori.com" 
+And user scrolled to the bottom of the page
+When user clicks a mailto button with an "Envelope" logo 
+Then a pop-up email window with fields "to", "cc", "subject" and "message body" is displayed
+And "career@quantori.com" address should appear in the "to" field automatically
+
+Scenario:1.2.3_12 Footer mailto button opens a new email window for a non-signed-in user
+Given user did not sign in to his default email client 
+And user opened "career.quantori.com" 
+And user scrolled to the bottom of the page
+When user clicks a mailto button with an "Envelope" logo 
+Then a pop-up email window with "Add an email account" is displayed
+And user should add his default email client 
```

Comment:
It sounds like an action, but not as expected result. I feel it is redundant because it is not related to application.

Reply:
Thank you for the comment!:)
I guess it is related to the user story. If I want to send an email via a pop-up window with my client and this opportunity is provided by the site, it should be tested

Reply:
In my understanding we don't need to test what is outside of our application:)
We need to test that email client is launched and data is passed. It is covered in Scenario:1.2.3_11.

Reply:
But if the user is not signed into his email-client, he will not be able to use this option. And it makes this button useless :)

Reply:
Agree with the Reviewer. 
Not necessary to test email clients.